### PR TITLE
Desktop version

### PR DESCRIPTION
**Pull request by glorymugizi**
Hello,
Using CSS styling I have made the desktop version look like the design given.
The media query used is for the Desktop screen: 768px or wider.

Thank you for reviewing my PR.

### CODE REVIEW

File: styles.css [hunk 1]
```diff
@@ -415,3 +432,243 @@ form {
...
+    margin-left: 20px;
+  }
+
+  .grid-item {
+    display: flex;
+    width: 62.5rem;
+    height: 25rem;
+    justify-content: space-between;
+    padding: 1.25rem;
+  }
```
Comment (binhussen) [inline styles.css]: - kindly, your cards need some padding, separate the description and images equally and ggive some space between texts.👌👍


| Your design                                                             | Figma design                                         |
| --------------------------------------------------------------------------------| --------------------------------------------------------- |
|||


😎️

File: styles.css [hunk 2]
```diff
@@ -415,3 +432,243 @@ form {
...
+    width: 100vw;
+
+    /* height: 646px; */
+  }
+
+  .contact-container {
+    display: flex;
+    flex-direction: column;
+    flex-wrap: wrap;
+    width: 40%;
```
Comment (binhussen) [inline styles.css]: - Kindly, center your contact form button and make the description as Figma design.👌👍


| Your design                                                             | Figma design                                         |
| --------------------------------------------------------------------------------| --------------------------------------------------------- |
|||


😎️

File: styles.css [hunk 3]
```diff
@@ -415,3 +432,243 @@ form {
...
+  .frame li {
+    padding-left: 10px;
+  }
+
+  /* About myself section */
+  #about-myself {
+    display: flex;
+    padding: 0;
+    border-top-right-radius: 100px;
+  }
```
Comment (binhussen) [inline styles.css:627]: - Good job on your about section 👍, I think it is better to add padding on the left and right to fit directly by the card direction👌.


| Your design                                                             | Figma design                                         |
| --------------------------------------------------------------------------------| --------------------------------------------------------- |
|||


😎️

**Review by binhussen — CHANGES_REQUESTED** [review]
Hi @glorymugizi , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!


## Highlights

- All linter checks passed ✔️
- Descriptive pull request ✔️
- Used GitHub flow ✔️

## Required Changes ♻️
- Kindly, stick to the [Figma design](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=39%3A122&t=rXsNjBPAGrehBgWL-0) as match as possible .👌

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: styles.css (hunk 2 above)
Comment (glorymugizi) [reply styles.css]: Hi@binhussen,
I aligned the button center to match the figma design.

Thank you.

File: styles.css (hunk 1 above)
Comment (glorymugizi) [reply styles.css]: Hi @binhussen,
Padding has been added on cards to match the design.

Thank you.

File: index.html [hunk 4]
```diff
@@ -12,172 +12,198 @@
...
           me.
         </p>
-        <p class="lets-connect"><a href="#">LET’S CONNECT</a></p>
+        <p class="lets-connect header-connect"><a href="#">LET’S CONNECT</a></p>
+      </div>
+      <div class="header-socios">
+        <ul class="social-media-links">
+          <li>
+            <a href="#"><img src="images/twitter.png" alt="twitter" /></a>
+          </li>
```
Comment (iambenkis) [inline index.html:43]: - [ ] You should stick to the design as much as possible (e.g. font, colours, images, text, margins) using [the templates in Figma](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=0%3A1). Kindly make sure that the `header` is centred as it is in the Figma design :+1: 

Your Design

Screenshot from 2022-12-10 16-29-46


Figma Design
Screenshot from 2022-12-10 16-33-38

**Review by iambenkis — APPROVED** [review]
Hi @glorymugizi , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights
Linters are passing 👍 
The project is running successfully 👍 
PR is well descriptive 👍 
## Optional suggestions

**_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._**

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me `@iambenkis` in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: styles.css (hunk 3 above)
Comment (glorymugizi) [reply styles.css:627]: Hi @iambenkis, The padding has been reduced to match the design.
Thank you for reviewing my PR.

**Pull request merged**